eng/ci/official-build.yml

```diff
@@ -46,6 +46,7 @@ extends:
          compiled:
            enabled: true
          runSourceLanguagesInSourceAnalysis: true
+         excludePathPatterns: "test/**"
```

Comment:
also saw following note in 1ES templates guidance - 

`Do not use * or ** at the end of the pattern.`

Reply:
updated the path pattern